### ISSUE TITLE
Extend CI to build a desktop release

### ISSUE THREAD

**Issue by holzeis** [itchysats/10101#133]

**Comment by holzeis** [comment]
What platforms should we support? for the Legends of Lightning Tournament?

**Comment by bonomat** [comment]
> What platforms should we support? for the Legends of Lightning Tournament?

If possible, I would aim for macOS, Linux and Windows. If Windows makes any troubles, I'd drop it immediately :)

**Comment by holzeis** [comment]
Should we still keep that ticket. If we really want to go for desktop I would rather build another UI for desktop instead of making it fit for desktop **and** mobile.

**Comment by klochowicz** [comment]
> Should we still keep that ticket? If we really want to go for desktop I would rather build another UI for desktop instead of making it fit for desktop **and** mobile.

Why would you make a different UI? this app is perfectly usable on desktop, and that blend seems to be direction of the UIs everywhere (e.g. just look at the new macOS - design language is taken straight out of ipadOS). given that we will likely not have many desktop users, I would just keep the same desktop.

**Comment by holzeis** [comment]
> > Should we still keep that ticket? If we really want to go for desktop I would rather build another UI for desktop instead of making it fit for desktop **and** mobile.
> 
> Why would you make a different UI? this app is perfectly usable on desktop, and that blend seems to be direction of the UIs everywhere (e.g. just look at the new macOS - design language is taken straight out of ipadOS). given that we will likely not have many desktop users, I would just keep the same UI.

fair enough, will be challenging though, and the app in its current design is IMHO not ready to be shipped to desktop. We designed primarily with mobile in mind and never thought about what to do to make this also work on desktop.

I still think having separate UIs for mobile and desktop makes our lives easier. but that's just my opinion - happy to take that up into a discussion.
